test/jdk/java/io/File/EmptyPath.java

```diff
@@ -221,11 +233,26 @@ public void list() throws IOException {
 
     @Test
     public void listFiles() throws IOException {
-        File child = new File(f.getAbsoluteFile(), "child");
+        listFiles(x -> x.listFiles());
+    }
+
+    @Test
+    public void listFilesFileFilter() throws IOException {
+        listFiles(x -> x.listFiles((FileFilter)null));
+    }
+
+    @Test
+    public void listFilesFilenameFilter() throws IOException {
+        listFiles(x -> x.listFiles((FilenameFilter)null));
+    }
+
```

Comment:
What would you think about test a non-null filter too?

Reply:
Add `listFiles` tests for non-null `FileFilter` and `FilenameFilter` in cfd494f.